.codeclimate.yml

```diff
@@ -0,0 +1,7 @@
+engines:
+  eslint:
+    enabled: false
+
+ratings:
+  paths:
+  - "**.js"
```

Comment:
Can this be indented another level? That's how I tend to write yml, but tbh I have no idea which is "right"